src/coreclr/tests/src/Interop/COM/NativeClients/Dispatch.csproj

```diff
@@ -4,7 +4,7 @@
     <IgnoreCoreCLRTestLibraryDependency>true</IgnoreCoreCLRTestLibraryDependency>
     <CLRTestScriptLocalCoreShim>true</CLRTestScriptLocalCoreShim>
     <RequiresMockHostPolicy>true</RequiresMockHostPolicy>
-    <IlrtTestKind>BuildOnly</IlrtTestKind>
+    <IlasmRoundTripIncompatible>true</IlasmRoundTripIncompatible>
```

Comment:
>IlasmRoundTripIncompatible [](start = 5, length = 26)

Is it incompatible because of  [https://github.com/dotnet/runtime/issues/11412](https://github.com/dotnet/runtime/issues/11412) ? If yes, may be add a comment in these files too?

Reply:
I'm not sure; it's possible. There wasn't a comment here before, and I was changing this case from "BuildOnly" which allows ildasm/ilasm round-trip, but doesn't actually run the result. I've decided not to support that mode of testing, so I just disabled that test entirely.